src/components/state-details.js

```diff
@@ -126,14 +126,19 @@ export default function StateDetailsPage ({ location, data }) {
     Cars_All: carsAll,
     EV_Registration: evRegistration,
   } = data.allVehiclesJson.edges[0].node
+  
   const pctEv = Math.round((evRegistration / carsAll) * 100 * 10) / 10
   const pctNonEv = Math.round((100 - pctEv) * 10) / 10
```

Comment:
These roundings are gross, I'd suggest doing `num.toFixed(2)`  which yields a string, and doing that to create an output string (if that's what we're trying to do).

Reply:
i didn't write this code but it looks to be accounting for Javascript's floating point inaccuracy. These are numbers used to calculate values and the strings are formatted below.